doc/nrf/gs_programming.rst

```diff
@@ -153,15 +154,15 @@ Complete the following steps to build |NCS| projects on the command line after c
 
          The Toolchain Manager dropdown menu options
 
-#.    Go to the specific sample or application directory.
-      For example, to build the :ref:`at_client_sample` sample, run the following command to navigate to the sample directory:
+#.    Go to the specific application directory.
+      For example, to build the :ref:`at_client_sample` sample, run the following command to navigate to the application directory:
```

Comment:
Not so sure about this as the directory name is "samples".
Maybe something like: ... navigate to its directory.

Reply:
I was going to do just that, but then second-guessed myself. I agree though, I think "its directory" is better. Adding the change now.